apps/reference/_supabase_dart/intro.mdx

```diff
@@ -6,6 +6,14 @@ sidebar_label: Supabase Dart Library
 
 # Supabase Dart Library
 
+:::note
+
+You're viewing the Supabase docs for the next version of our library which is not yet released.
+
+Refer to the `v0` docs for a stable release.
```

Comment:
Great point. supabase-flutter library is in a bit of a weird state right now, because there is the v0, which is the "stable" release with no breaking changes, and there is the v1.0 developer preview. 

Maybe we can say something like this instead?

```
You're viewing the Supabase docs for a developer preview version.

Refer to the `v0` docs for the previous release.
```

Reply:
Done!